src/Credentials/ExternalAccountCredentials.php

```diff
@@ -150,11 +152,6 @@ private static function buildCredentialSource(array $jsonKey): ExternalAccountCr
                     'The regional_cred_verification_url field is required for aws1 credential source.'
                 );
             }
-            if (!array_key_exists('audience', $jsonKey)) {
```

Comment:
was this intentionally removed?

Reply:
Yes because we already verify that the "audience" key exists [earlier in this function](https://github.com/googleapis/google-auth-library-php/pull/525/files#diff-2826fe12b39f9080278b434e47cda5ff572f3e846b3f1d4e043feb39e91f9c5fR83-R87)